src/_P109_ThermOLED.ino

```diff
@@ -891,11 +862,11 @@ void P109_setSetpoint(String sptemp) {
   P109_display_setpoint_temp(0);
 }
 
-void P109_setHeatRelay(byte state) {
+void P109_setHeatRelay(const uint8_t& state) {
   uint8_t relaypin = Settings.TaskDevicePluginConfig[Plugin_109_taskindex][4];
 
   if (loglevelActiveFor(LOG_LEVEL_INFO)) {
-    String  logstr   = F("Thermo : Set Relay");
+    String logstr = F("Thermo : Set Relay");
 
     logstr += relaypin;
     logstr += F("=");
```

Comment:
These kinda things could also be changed to adding a char instead of a flash string